src/tokens.ts

```diff
@@ -1,13 +1,14 @@
 import { SnsWasmCanister } from '@dfinity/nns';
 import axios from 'axios';
-import TokensJson from './tokenlist.json';
-import TestnetTokensJson from './tokenlist.testnet.json';
 import { HttpAgent } from '@dfinity/agent';
 import { initSnsWrapper } from '@dfinity/sns';
 import { Principal } from '@dfinity/principal';
 import { isDefined } from './utils';
 
 const IC_API_BASE_URL = 'https://ic-api.internetcomputer.org';
+export const TOKEN_LIST_BASE_URL = "https://raw.githubusercontent.com/infinity-swap/token-lists/main/src"
+export const TOKENS_JSON_LIST_URL = `${TOKEN_LIST_BASE_URL}/tokenlist.json`
+export const TEXT_TOKENS_JSON_LIST_URL = `${TOKEN_LIST_BASE_URL}/tokenlist.testnet.json`
```

Comment:
I think it should be `TEST_TOKENS_JSON_LIST_URL` or maybe `TOKENS_ TEST_JSON_LIST_URL`